sssd_test_framework/utils/sssd.py

```diff
@@ -783,3 +783,50 @@ def autofs(self) -> None:
         """
         self.sssd.authselect.select("sssd")
         self.sssd.enable_responder("autofs")
+
+    def proxy(self,
+              proxy: str,
+              provider: str | list[str] = 'id',
+              role: MultihostRole = None,
+              domain: str | None = None,):
+        """
+        :param domain: Existing domain name, defaults to None (= default domain)
+        :type domain: str | None, optional
```

Comment:
Move to the end. Order of parameters here must follow the parameters of the function.

This docstring is used to render documenation.